src/components/input/TextField/TextField.tsx

```diff
@@ -57,9 +58,21 @@ const TextField = ({
   tooltipMsg,
   prefix,
   tooltipIcon,
+  rightIcon,
   ...props
 }: TextFieldProps) => {
   const { singleBorder, doubleBorder, prefixStyles, icon } = useStyles()
+
+  const handleAdornment = () => {
+    if (rightIcon) {
+      return rightIcon
+    }
+    if (prefix) {
+      return <Typography className={prefixStyles}>{prefix}</Typography>
+    }
+    return ""
```

Comment:
This all looks great, but I'm just wondering if this should maybe return `null` instead of empty string. Either way, I'm sure it's fine.